girder/models/model_base.py

```diff
@@ -684,11 +717,10 @@ def _hasUserAccess(self, perms, userId, level):
                 return True
         return False
 
-    def _setAccess(self, doc, id, entity, level, save):
+    def _setAccess(self, doc, id, entity, level, save, flags=None, user=None):
```

Comment:
by the same logic that `setPublicFlags` has a `force` param, this method should too